kdb+.html.markdown

```diff
@@ -6,17 +6,17 @@ contributors:
 filename: learnkdb.q
 ---
 
-The q langauge and its database component kdb+ were developed by Arthur Whitney
-and released by Kx systems in 2003. q is a descendant of APL and as such is
-very terse and a little strange looking for anyone from a "C heritage" language
-background. Its expressiveness and vector oriented nature make it well suited
+The q language and its database component kdb+ were developed by Arthur Whitney
+and released by [Kx Systems](http://kx.com) in 2003. q is a descendant of APL and as such is
+very terse and a little strange-looking for anyone from a "C heritage" language
+background. Its expressiveness and vector-oriented nature suit it
 to performing complex calculations on large amounts of data (while also
 encouraging some amount of [code
 golf](https://en.wikipedia.org/wiki/Code_golf)). The fundamental structure in
 the language is not the object but instead the list, and tables are built as
-collections of lists. This means - unlike most traditional RDBMS systems -
-tables are column oriented.  The language has both an in-memory and on-disk
-database built in, giving a large amount of flexibility. kdb+ is most widely
+collections of lists. This means &ndash; unlike most traditional RDBMS systems &ndash;
```

Comment:
Why did you change them to HTML entities instead of using the actual character? If you're changing it from a normal '-' symbol another please use the actual character not the HTML entity. Also please use the mdash `—` to separate a section of text — ndashes is for ranges of things not for a break in the sentence.